src/archivematicaCommon/lib/elasticSearchFunctions.py

```diff
@@ -234,74 +239,91 @@ def _sortable_string_field_specification(field_name):
         }
     }
 
-def set_up_mapping(conn, index):
-    machine_readable_field_spec = {
-        'type':  'string',
-        'index': 'not_analyzed'
+def set_up_mapping_aip_index(conn):
+    LOGGER.info('Creating AIP mapping...')
+    conn.indices.put_mapping(doc_type='aip', body={'aip': {'date_detection': True}}, index='aips')
+    LOGGER.info('AIP mapping created.')
+
+    mapping = {
+        'name': _sortable_string_field_specification('name'),
+        'size': {'type': 'double'},
+        'uuid': MACHINE_READABLE_FIELD_SPEC
     }
 
-    if index == 'transfers':
-        mapping = {
-            'filename'     : {'type': 'string'},
-            'relative_path': {'type': 'string'},
-            'fileuuid'     : machine_readable_field_spec,
-            'sipuuid'      : machine_readable_field_spec,
-            'accessionid'  : machine_readable_field_spec,
-            'status'       : machine_readable_field_spec,
-            'origin'       : machine_readable_field_spec,
-            'ingestdate'   : {'type': 'date' , 'format': 'dateOptionalTime'},
-            'created'      : {'type': 'double'},
-            'size'         : {'type': 'double'},
-            'tags'         : machine_readable_field_spec,
-            'file_extension': machine_readable_field_spec,
-            'bulk_extractor_reports': machine_readable_field_spec,
-            'format'       : {
-                'type'      : 'nested',
-                'properties': {
-                    'puid'  : machine_readable_field_spec,
-                    'format': {'type': 'string'},
-                    'group' : {'type': 'string'},
-                }
+    LOGGER.info('Creating AIP mapping...')
+    conn.indices.put_mapping(
+        doc_type='aip',
+        body={'aip': {'date_detection': True, 'properties': mapping}},
```

Comment:
Nice catch, it was apparently like this before too and I just didn't notice it when I was refactoring/moving code around. I don't think we'll need the first create mapping call... I'll remove it and find out